src/modules/fancyzones/FancyZonesLib/Resources.resx

```diff
@@ -254,4 +254,7 @@
   <data name="Setting_Description_FlashZonesOnQuickSwitch" xml:space="preserve">
     <value>Flash zones when switching layout</value>
   </data>
+  <data name="Setting_Description_Custom_Colors" xml:space="preserve">
+    <value>Use custom colors</value>
```

Comment:
Better would be "Use custom zone colors"

Reply:
See @niels9001 suggestion https://github.com/microsoft/PowerToys/pull/14158#discussion_r739792102